TarSCM/__init__.py

```diff
@@ -31,8 +28,8 @@ def run():
 
     try:
         task_list.process_list()
-    except OptionsError as e:
-        print(e)
+    except OptionsError as exc:
```

Comment:
Because pylint otherwise is complaining about invalid-name. pylint wants variable names to be at least 3 char long.